proposals/low-level-struct-improvements.md

```diff
@@ -157,7 +157,7 @@ Next the rules for ref reassignment need to be adjusted for the presence of `ref
 The left operand of the `= ref` operator must be an expression that binds to a ref local variable, a ref parameter (other than `this`), an out parameter, **or a ref field**.
 
 > For a ref reassignment in the form ...
-> 1. `x.e1 = ref e2`: where `x` is *safe-to-escape* to *calling method* then `e2` must be *ref-safe-to-escape* to the *calling method*
+> 1. `x.e1 = ref e2`: where `x` is *safe-to-escape* at least *return only* then `e2` must have at least the same *ref-safe-to-escape* as `x`.
```

Comment:
>have at least the same *ref-safe-to-escape* as `x`. [](http://example.com/codeflow?start=90&length=51)

Perhaps "must have *ref-safe-to-escape* at least as large as `x`", to match the second bullet, and so it's clear that larger is allowed rather than smaller.